lib/jsonclient.rb

```diff
@@ -37,7 +37,7 @@ def request(method, uri, *args, &block)
 
   def argument_to_hash_for_json(args)
     hash = argument_to_hash(args, :body, :header, :follow_redirect)
-    if hash[:body].is_a?(Hash)
+    if hash[:body].is_a?(Hash) || hash[:body].is_a?(Array)
```

Comment:
@nahi What exactly is the point of restricting `body` to be a Hash (or an Array after this PR)? Seems like a strange thing.